public/stylesheets/style.css

```diff
@@ -38,6 +38,46 @@ a {
 .rb-white {
   color: #ffffff; 
 }
+.g-badge {  
+  display: inline-block;
+  min-width: 10px;
+  padding: 1px 2px;
+  font-size: 12px;
+  font-weight: 700;
+  line-height: 1;  
+  text-align: center;
+  white-space: nowrap;
+  vertical-align: baseline;
+  border-radius: 50%;
+  display: inline-block;
+  width: 25px;
+  height: 25px;
+  margin: 0 2px 0 2px;    
+  content: url(http://cdn.mysitemyway.com/etc-mysitemyway/icons/legacy-previews/icons/magic-marker-icons-culture/115276-magic-marker-icon-culture-crown-five-points2.png) no-repeat 0 0;
+  background-size: 100%;
+  background-color: #fff;
+  vertical-align: -7px;
+}
+.badge-roster {
+
+}
+.badge-leaders {
+
+}
+.badge-goats {
+
+}
+.goats-icon {
+  content: url(http://upload.wikimedia.org/wikipedia/en/thumb/c/c0/Portuguese_Ibex_Cabrera_Goat.svg/500px-Portuguese_Ibex_Cabrera_Goat.svg.png);
```

Comment:
they are just placeholding for now. I haven't decided on the final ones. So once we do I'll host the ones we pick.